models/eqV2_tracegrad/save_geo_opt_file.py

```
@@ -0,0 +1,51 @@
+        # 行分隔JSON通常以 { 开头
+        try:
+            f.seek(0)
+            line = f.readline().strip()
+            json.loads(line)  # 验证是否为完整JSON对象
+            return "lines"
+        except json.JSONDecodeError:
+            return "unknown"
+
+
+# 原始文件路径
+ml_relaxed_structs_path = "/data/fywang/code/matbench-discovery/models/eqV2_tracegrad/eqV2-small-dens-tracegrad.json.gz"
```

Ruff (0.12.2)

Line too long (120 > 88)

(E501)